httprange/itemkind_string.go

```diff
@@ -0,0 +1,16 @@
+// Code generated by "stringer -type=itemKind lex.go"; DO NOT EDIT
+
+package httprange
+
+import "fmt"
+
+const _itemKind_name = "itemErroritemUnititemRespitemEOFitemStartitemEnditemLength"
+
+var _itemKind_index = [...]uint8{0, 9, 17, 25, 32, 41, 48, 58}
+
+func (i itemKind) String() string {
+	if i < 0 || i >= itemKind(len(_itemKind_index)-1) {
+		return fmt.Sprintf("itemKind(%d)", i)
+	}
+	return _itemKind_name[_itemKind_index[i]:_itemKind_index[i+1]]
```

Comment:
Is there a way to make `stringer` not output underscore-prefixed var names? Only asking, since it makes `golint` complain:

```
itemkind_string.go:7:7: don't use underscores in Go names; const _itemKind_name should be _itemKindName
itemkind_string.go:9:5: don't use underscores in Go names; var _itemKind_index should be _itemKindIndex
```